cf/crypto.m4

```diff
@@ -135,6 +135,17 @@ if test "$crypto_lib" = "unknown"; then
 
 fi
 
+AC_ARG_WITH(pkcs11-module,
```

Comment:
Yes, but also hcrypto does not link against krb5 (circular dependency).

Sent from my iPhone

> On 29 Nov 2015, at 10:36, Nico Williams <notifications@github.com> wrote:
> 
> In cf/crypto.m4:
> 
> > @@ -135,6 +135,17 @@ if test "$crypto_lib" = "unknown"; then
> >  
> >  fi
> >  
> > +AC_ARG_WITH(pkcs11-module,
> Why shouldn't this just be a configuration parameter in krb5.conf? Why bother with autoconf'ing this? My guess: to provide a platform-specific/sane default?
> 
> —
> Reply to this email directly or view it on GitHub.
>